src/Illuminate/Support/Testing/Fakes/QueueFake.php

```diff
@@ -323,6 +331,10 @@ public function push($job, $data = '', $queue = null)
      */
     public function shouldFakeJob($job)
```

Comment:
Shouldn't this method be `protected`? I created `shouldDispatchJob` as public to be consistent with this one, let me know what's the correct approach here.